apis/datadoghq/v2alpha1/test/builder.go

```diff
@@ -770,14 +770,21 @@ func (builder *DatadogAgentBuilder) WithOriginDetectionUnified(enabled bool) *Da
 	return builder
 }
 
-// Global OriginDetectionUnified
+// Global Registry
```

Comment:
This is un-related to the PR, is a minor comment fix from a copy-paste in https://github.com/DataDog/datadog-operator/commit/021bd60d25e61e8ab2aa10a77e5894c6c0690246#diff-79f1664318b07c82e329953388426262e2bf32b1a831bc4413e12c77c6416a2aR695-R702